openhtf/io/output/print_ToConsole.py

```diff
@@ -0,0 +1,93 @@
+"""Module to display test summary on console."""
+import os
+import sys
+from openhtf import util
+from openhtf.io import test_record as tr
+from openhtf.util import measurements as mea
```

Comment:
I generally try not to rename imported modules because I feel like doing so hurts readability. Do the other contributors (@grybmadsci, @fahhem)  have thoughts about whether/when we should do this?